ocp_resources/task_run.py

```diff
@@ -0,0 +1,59 @@
+# API reference: https://tekton.dev/docs/pipelines/taskruns/
+
+from ocp_resources.resource import NamespacedResource
+
+
+class TaskRun(NamespacedResource):
+    api_group = NamespacedResource.ApiGroup.TEKTON_DEV
+
+    def __init__(
+        self,
+        task_ref=None,
+        task_spec=None,
+        params=None,
+        service_account_name=None,
+        taskrun_timeout=None,
+        **kwargs,
+    ):
+        """
+        Args:
+            task_ref (str, mandatory): Base task to run taskrun
+            task_spec (str, mandatory): Base task to run taskrun
```

Comment:
can be either `task_ref` or `task_spec` 
1. please update docstring
2. what happens if both are provided? is this supported? if not, please check and raise

Reply:
ack . It wll throw error : admission webhook "validation.webhook.pipeline.tekton.dev" denied the request: validation failed: expected exactly one, got both